config/app.default.php

```diff
@@ -84,6 +85,7 @@
         'default' => [
             'className' => 'File',
             'path' => CACHE,
+            'url' => env('CACHE_DEFAULT_URL', ''),
```

Comment:
The cache URLs should be consistent ans transparent in naming like `CACHE_DEFAULT_URL`, and `CACHE_PERSISTENT_URL` and `CACHE_MODELS_URL`.